tests/home_robot_sim/test_habitat_env.py

```diff
@@ -2,8 +2,14 @@
 from pathlib import Path
 from subprocess import Popen
 
-import habitat
 import pytest
+
+try:
+    import habitat
```

Comment:
I can remove this if desired -- I don't have habitat installed, and I'm not sure the usage of the repo in general requires it? So I made this optional....

Reply:
Yeah would be great if you can remove this, no worries if it doesn't pass locally since everything will ultimately be run on CircleCI.